### PR TITLE
HTTP logger & UI Changes

### PR DESCRIPTION
**Pull request by Hydra-Source**

### CODE REVIEW

**Comment by Hydra-Source** [comment]
Changed PR's main concept to UI overhaul & improvements.
image

**Comment by Hydra-Source** [comment]
Closes #30

**Comment by tyackman** [comment]
Styling looks pretty heckin nice imo

File: src/game/pointers/Pointers.cpp [hunk 1]
```diff
@@ -94,6 +94,13 @@ namespace YimMenu
 			LOG(INFO) << "HWND: " << Hwnd;
 		});
 
+        constexpr auto networkRequest = Pattern<"4C 8B DC 49 89 5B 08 49 89 6B 10 49 89 73 18 57 48 81 EC ? ? ? ? 48 8B 01">("NetworkRequest");
```
Comment (DayibBaba) [inline src/game/pointers/Pointers.cpp:97]: All the other pointers are followed by `Ptrn` in their names.

**Comment by Hydra-Source** [comment]
Indents are fucked thanks to github

**Comment by Hydra-Source** [comment]
> > Indents are fucked thanks to github
> 
> That's a feature that intentionally reveals if you've mixed tabs and spaces in a file

image

File: src/game/frontend/manager/Widgets/Button.cpp [hunk 2]
```diff
@@ -3,11 +3,11 @@
 
 namespace YimMenu
 {
-	Button::Button(const std::string_view& name, std::function<void()> onClick, const ImVec2 size, const std::string_view& information) :
-	    m_Name(name),
-	    m_Information(information),
-	    m_OnClick(onClick),
-	    m_Size(size)
+	Button::Button(const std::string_view& Name, std::function<void()> OnClick, const ImVec2 Size, const std::string_view& Information) :
```
[the remark this answers is not in the document]
Comment (Hydra-Source) [reply src/game/frontend/manager/Widgets/Button.cpp:6]: Didn't get that info, sure.

**Comment by Hydra-Source** [comment]
> @Hydra-Source Can you try to load the menu with the Vulkan renderer?

Vulkan on AMD is broken at the moment

**Comment by Hydra-Source** [comment]
> > > @Hydra-Source Can you try to load the menu with the Vulkan renderer?
> > 
> > 
> > Vulkan on AMD is broken at the moment(was always) use DX12 it should be better now and clear your dx cache and try.
> 
> It worked fine for me but it now crashes on the font thing (using Nvidia)

Oh really, i couldn't test since i was on AMD but sure thing i'll look into it tonight with my laptop. Thanks for letting me knowing.

**Pull request merged**